pkg/nsx/cluster.go

```diff
@@ -31,6 +31,12 @@ const (
 	GREEN
 )
 
+var typeToString = map[ClusterHealth]string{
+	RED:    "RED",
+	ORANGE: "ORANGE",
+	GREEN:  "GREEN",
+}
```

Comment:
Can we merge ClusterHealth and typeToString together? Let we just use string for health return value